src/testHelpers/withTimeMachine.jsx

```diff
@@ -1,21 +1,22 @@
-import React, { useEffect } from 'react';
+import { useEffect, useState } from 'react';
 import { element } from 'prop-types';
 import {
   startTimeMachine,
   resetTimeMachine,
 } from '../../.storybook/time-machine';
 // This affects the global Date object for the storybook application, to ensure consistency in chromaticQA testing.
 const WithTimeMachine = ({ children }) => {
-  // set time, but leave it ticking, so knobs work
-  startTimeMachine(false);
-  // setting tick to false, prevents the timemachine from ticking,
-  // so the timestamp displayed is frozen, and there are no side effects.
+  const [componentToRender, setComponentToRender] = useState(null);
+
   useEffect(() => {
-    resetTimeMachine();
-    startTimeMachine(true);
-    return resetTimeMachine();
+    function handleStateChange() {
```

Comment:
Do we need this function? Instead we could just put `setComponentToRender(children);` below